specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2018-02-01-preview/containerregistry_build.json

```diff
@@ -0,0 +1,2220 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-02-01-preview",
+    "title": "ContainerRegistryManagementClient"
+  },
+  "host": "management.azure.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/buildDefinitions": {
+      "get": {
+        "tags": [
+          "BuildDefinitions"
+        ],
+        "summary": "Lists all the build definitions for a specified container registry.",
+        "operationId": "BuildDefinitions_List",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
```

Comment:
I believe you can remove 
```
        "consumes": [],
        "produces": [
          "application/json"
        ],
```
from each operation as they are already set globally.

Reply:
Same for other cases.